python/nammu/view/ToolbarView.py

```diff
@@ -7,91 +7,163 @@
 '''
 
 from javax.swing import JToolBar, ImageIcon, JButton
+from java.lang import ClassLoader
 
 class ToolbarView(JToolBar):
     
     def __init__(self, controller):
-        print "I'm the toolbar view"
         
         #Give reference to controller to delegate action response
         self.controller = controller
         
-        newIcon = ImageIcon("../../../resources/images/new.png")
-        newFileButton = JButton("New", newIcon, 
+        #Build Toolbar following schema in issue#18 and mockups in
+        #https://github.com/UCL-RITS/nammu/tree/master/doc/mockups
+        #https://github.com/UCL-RITS/nammu/issues/18
+        
+        #TODO Refactor to avoid duplication - See issue#16 
+        #https://github.com/UCL-RITS/nammu/issues/16
+        
+        #Create helper object to load icon images in jar
+        loader = ClassLoader.getSystemClassLoader()
+        
+        newIcon = ImageIcon(loader.getResource("resources/images/new.png"))
+        newFileButton = JButton(newIcon, 
                                 actionPerformed=self.onNewFileClick)
+        newFileButton.setToolTipText("Creates empty ATF file for edition")
         self.add(newFileButton)
         
-        openIcon = ImageIcon("../../../resources/images/open.png")
-        openFileButton = JButton("Open", openIcon, 
+        openIcon = ImageIcon(loader.getResource("resources/images/open.png"))
+        openFileButton = JButton(openIcon, 
                                  actionPerformed=self.onOpenFileClick)
+        openFileButton.setToolTipText("Opens ATF file for edition")
         self.add(openFileButton)
         
-        saveIcon = ImageIcon("../../../resources/images/save.png")
-        saveFileButton = JButton("Save", saveIcon, 
+        saveIcon = ImageIcon(loader.getResource("resources/images/save.png"))
+        saveFileButton = JButton(saveIcon, 
                                  actionPerformed=self.onSaveFileClick)
+        
+        saveFileButton.setToolTipText("Saves current file")
         self.add(saveFileButton)
         
-        closeIcon = ImageIcon("../../../resources/images/close.png")
-        closeFileButton = JButton("Close", closeIcon, 
+        closeIcon = ImageIcon(loader.getResource("resources/images/close.png"))
+        closeFileButton = JButton(closeIcon, 
                                   actionPerformed=self.onCloseFileClick)
+        closeFileButton.setToolTipText("Close current file")
         self.add(closeFileButton)
+        
+        printIcon = ImageIcon(loader.getResource("resources/images/print.png"))
```

Comment:
Metaprogram as above so all the icons can be controlled by a data structure to avoid copypasta.